ingestion/src/metadata/ingestion/source/database/redshift.py

```diff
@@ -460,6 +468,36 @@ def create(cls, config_dict, metadata_config: OpenMetadataConnection):
             )
         return cls(config, metadata_config)
 
+    def get_table_type(self, schema_name: str, table_name: str) -> TableType:
+        """
+        Fetch the table types and return standard TableType
+        """
+        tabel_fqn = fqn.build(
+            self.metadata,
+            Table,
+            database_name=self.context.database.name.__root__,
+            schema_name=schema_name,
+            table_name=table_name,
+            service_name=self.config.serviceName,
+        )
+        if not self.table_type_map:
+            table_type_query = "SELECT database_name,schema_name, table_name,table_type FROM svv_all_tables"
+            result = self.engine.execute(table_type_query)
+            self.table_type_map = {
+                fqn.build(
+                    self.metadata,
+                    Table,
+                    database_name=row[0],
+                    schema_name=row[1],
+                    table_name=row[2],
+                    service_name=self.config.serviceName,
+                ): row[3]
+                for row in result
```

Comment:
From what I understand (and testing locally) `row` will be a `sqlalchemy.engine.row.LegacyRow` type. We sh🙂ould be able to be explicit when fetching the value of the row (e.g. `row.table_name`, etc.) vs accessing the value via an index.